action.yml

```
@@ -168,21 +163,141 @@ runs:
           git checkout $FROM_BRANCH
           git checkout $INTO_BRANCH
 
+          # Determine if we will rebuild dist file during merge-up
+          # This is based simply on if there are changes in the client/ directory
+          REBUILD=
```

```suggestion
          REBUILD=0
```
We've established a bit of a pattern of starting off with a default value - which will be useful with `echo "REBUILD is $REBUILD"` as it will say "REBUILD is 0" instead of "REBUILD is" - the latter of which makes it look like something went wrong.

---

Updated